src/Defs.h

```diff
@@ -385,10 +385,10 @@ extern TicksTime g_cycleStartTime;
 extern debug_level_t g_debug_level;
 
 #ifdef  USING_VMA_EXTRA_API
-extern unsigned char* g_pkt_buf;
-extern struct vma_packets_t* g_pkts;
-extern unsigned int g_pkt_index;
-extern unsigned int g_pkt_offset;
+extern __thread unsigned char* g_pkt_buf;
```

Comment:
once using the '__thread' directive it would be better to:
1.  use MACRO definition. I'm guessing this will translate into different API's for different OS's
2.  move all parameters into a single structure for mapping VMA_EXTAR variable. Bringing the thread specific data into context has cost, let's get hit once for all 4 members